Documentation/helm-ceph-cluster.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Ceph Cluster
+weight: 10200
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+# Ceph Cluster Helm Chart
+
+Installs a [Ceph](https://ceph.io/) cluster on Rook using the [Helm](https://helm.sh) package manager.
+
+## Warning
+
+This chart is released to the community for testing but is considered in development. It is currently only available on the `master` channel.
+Once released, the repository should be updated to the `release` channel if you wish to switch to the stable branch.
+
+## Prerequisites
+
+* Kubernetes 1.13+
+* Helm 3.x
+* Preinstalled Rook Operator. See the [Helm Operator](https://rook.github.io/docs/rook/master/helm-operator.html) topic to install.
+
+## Installing
+
+The `helm install` command deploys rook on the Kubernetes cluster in the default configuration.
+The [configuration](#configuration) section lists the parameters that can be configured during installation. It is
+recommended that the rook operator be installed into the `rook-ceph` namespace (you will install your clusters into
+separate namespaces).
+
+If the operator was installed in a non-default location, the namespace of the *Rook Operator* installed
+must be set in the `operatorNamespace` variable.
+
+Rook currently publishes builds of this chart to the `master` channel.
+
+### Master Channel
+
+The master channel is the most recent release of Rook that includes in development features.
+
+```console
+helm repo add rook-release https://charts.rook.io/master
+helm install --create-namespace --namespace rook-ceph rook-ceph --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster
+```
+
+### Development Build
```

Comment:
Let's remove this section for this experimental documentation, and just focus on the master channel.